### ISSUE TITLE
Improve muxing with turtlebot_teleop

### ISSUE THREAD

**Issue by bit-pirate** [turtlebot/turtlebot#105]
Seems like turtlebot_teleop does not work well with the velocity muxer using the default settings, because it is constantly publishing velocity commands.

For example, one cannot run turtlebot's teleop together with navigation, see [this question](http://answers.ros.org/question/65626/autonomous-navigation-of-a-known-map-with-turtlebot-problem/?answer=66283#post-id-66283). Using navigation together with kobuki's teleop works fine.

I'm not sure, if this is just a matter of changing the defaults or we need to touch the turtlebot_teleop code (maybe even dropping it?).

**Comment by corot** [comment]
Yep, we need to modify turtlebot teleop; cmd_vel_mux needs that all inputs behave politely to work, no spaming with zero-velocity messages when inactive.
